velox/docs/configs.rst

```diff
@@ -436,6 +441,16 @@ Each query can override the config by setting corresponding query session proper
      - string
      - 10MB
      - Maximum bytes for sort writer in one batch of output. This is to limit the memory usage of sort writer.
+   * - file-preload-threshold
+     -
+     - integer
+     - 1MB
+     - Usually Velox fetches the meta data firstly then fetch the rest of file. But if the file is very small, Velox can fetch the whole file directly to avoid multiple IO requests. The parameter control the threshold when whole file is fetched. 
+   * - directory-size-guess
+     -
+     - integer
+     - 8MB
+     - Define the data size which is fetched with meta data together to void multiple IO requests. It's useful in file format where the immediately following data needs to be fetched after meta data
```

Comment:
Can you give an example of such a file format? AFAIK both DWRF/ORC and Parquet have metadata after data.

Reply:
@zhenghaozzz is it for parquet only?

Reply:
Both Dwrf reader and Parquet reader

Reply:
Do you mean for both dwrf and parquet, we need to fetch the immediate following data after meta data? Do you know what's the data?